POSTINSTALL-README.md

```diff
@@ -127,6 +127,11 @@ To validate your secureblue setup, run:
 ```
 ujust audit-secureblue
 ```
+
+## Optional: `hardened-chromium` Flags
+The included hardened-chromium browser has some additional settings in `chrome://flags` you *may* want to set for additional hardening, and convenience. (At the cost of *some* functionality.)
```

Comment:
> (At the cost of *some* functionality.)

Maybe "That can cause functionality issues in some cases"

Reply:
I agree that's better wording, committed the change